types/wlr_surface.c

```diff
@@ -929,3 +929,17 @@ void wlr_surface_send_leave(struct wlr_surface *surface,
 		}
 	}
 }
+
+static inline int64_t timespec_to_msec(const struct timespec *a) {
```

Comment:
We have this functions in a couple of places.
It should probably live in util or something.

Reply:
Actually, there's only one of them in wlroots, which is this one. All others are in `examples/` (in `output-layout.c`, `rotation.c` and `simple.c`), so I wonder if it's worth it.